scripts/pre_commit.sh

```diff
@@ -8,8 +8,8 @@ isort .
 black src tests
 pre-commit run --all-files
 bandit -r src -ll
-pytest -m fast
+pytest -m fast --cov=src/ --cov-report html
```

Comment:
Is it possible we can combine this with .coveragerc file we have which omits the protos?

Reply:
Or possibly the settings in setup.cfg for pytest-cov, and then we can add the htmlcov dir to .gitignore?
```
[tool:pytest]
# Options for py.test:
# Specify command line options as you would do when invoking py.test directly.
# e.g. --cov-report html (or xml) for html/xml output or --junitxml junit.xml
# in order to write a coverage file that can be read by Jenkins.
# TODO (gmuraru) change cov-fail-under as soon as possible
addopts =
    --cov syft --cov-report term-missing --cov-fail-under 82 --no-cov-on-fail
    --verbose
norecursedirs =
    dist
    build
    .tox
testpaths = tests
```